astropy/utils/compat/numpy/lib/stride_tricks.py

```diff
@@ -73,7 +73,7 @@ def broadcast_arrays(*args, **kwargs):
 
         Parameters
         ----------
-        `*args` : array_likes
+        `*args` : array-likes
```

Comment:
I would *not* do this single one, since here we just copy a more recent routine from numpy, so that it can also be used with earlier numpy versions. This copy should stay the same as in numpy proper.  Otherwise, +1.